nlopt/config - win.h

```diff
@@ -0,0 +1,116 @@
+/* config.h.  Generated from config.h.in by configure.  */
```

Comment:
This should be removed from patch & develop as it's developer-specific and should be generated individually by the developer by `configure` during the build process.